frontend/src/app/modules/home/home-page/home-page.component.ts

```diff
@@ -26,6 +28,7 @@ export class HomeComponent implements OnInit, OnDestroy {
     ) { }
 
     async ngOnInit() {
+        this.authorizedUser.firstName = 'Test';
```

Comment:
Please use { ...obj } for this

Reply:
this.authorizedUser = new {firstname: 'Test', ...} as User;
It's okay?